<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Question_1_3</class>
 <widget class="QDialog" name="Question_1_3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Change_Label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>241</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>Change 1 Label!</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>269</width>
     <height>58</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>269</width>
     <height>79</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>269</width>
     <height>122</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>269</width>
     <height>249</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
